crates/biome_js_analyze/src/lint/nursery/use_explicit_function_return_type.rs

```diff
@@ -128,6 +130,14 @@ impl Rule for UseExplicitFunctionReturnType {
     type Options = ();
 
     fn run(ctx: &RuleContext<Self>) -> Self::Signals {
+        let source_type = ctx.source_type::<JsFileSource>().language();
+        let is_ts_source = source_type.is_typescript();
+        let is_declaration = source_type.is_definition_file();
+
+        if is_declaration || !is_ts_source {
```

Comment:
I think the following code is enough clear, we don't have to use intermediate variable.
```suggestion
        if !source_type.is_typescript() || source_type.is_definition_file() {
```

Reply:
I’ve updated the code accordingly. Thanks for the suggestion!